changelog/unreleased/features/2877-2904-2907--pipeline-string-parsing.md

```diff
@@ -0,0 +1,6 @@
+The `export`, `import` and `spawn source` commands now support an optional
```

Comment:
I wouldn't mention the `spawn source` command here; it's not documented currently, and I don't think that we will  have it anymore after introducing the `from` operator. Mentioning it here probably adds more confusion than it helps advertise the functionality that we actually want to show off, which is pipelines for import and export.

To be clear: The code for it can and should stay, after all it's the same code path that the source also uses.

Reply:
I smoothed out the documentation in https://github.com/tenzir/vast/pull/2907/commits/b63d88bc14d060cdc7f7d08af85710ea62c14f53